docs/quick_start/installing.rst

```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
     }
 
 Now, it's time to restart Fledge. Thereafter you will see Fledge is running with PostgreSQL.
+
+
+Using Docker Containerizer to install Fledge
+#############################################
+
+Fledge is provided a private registry. No auth/TLS yet on this private registry for docker container.
```

Comment:
Fledge Docker containers are provided in a private repository. This repository has no authentication or encryption associated with it.